### ISSUE TITLE
[5.0.3] Articles in backend not visible

### ISSUE THREAD

**Issue by herzogj** [joomla/joomla-cms#43163]
### Steps to reproduce the issue
Im logged in as a super user to a brand new Joomla 5.0.3 installation (with a template). I'm the only user and I can't see the articles in the backend. They show up in smart search, in the database and also on the frontend, but after creation in the backend I can't see them (and do some changes for example). No filter is set.

Some of the articles have been 'imported' during quickstart from the template but others were created with the New button in Content -> Articles. No DB imports were made.

What is wrong here? 

### Expected result
See all articles and modify them.

### Actual result
"No Matching Results" in Content -> Articles.

### System information (as much as possible)
Joomla 5.0.3
PHP 8.3.2
MariaDB 10.6.14

**Comment by brianteeman** [comment]
The problem is from the "import" - they almost certainly dont have an entry in the workflow table. You should contact the provider of the template

**Comment by herzogj** [comment]
Actually, I do have entries in the table workflow_associations. I also deleted all of them and recreated the inserts with
`INSERT INTO npfrq_workflow_associations (item_id, stage_id, extension) 
SELECT c.id as item_id, '1', 'com_content.article' FROM npfrq_content AS c 
WHERE NOT EXISTS (SELECT wa.item_id FROM npfrq_workflow_associations AS wa WHERE wa.item_id = c.id);`

If one association would be missing, would the backend show nothing? Because I created articles by hand in the backend and they are also not visible.
